subsys/bluetooth/audio/tbs.c

```diff
@@ -628,6 +632,73 @@ static void net_buf_put_current_calls(const struct tbs_inst *inst, struct net_bu
 	}
 }
 
+static bool is_tbs_uri_unique(const char *uri_list, const char *uri_to_search)
+{
+	bool is_matched = false;
+	size_t uri_len = strlen(uri_to_search);
+	char *uri_pos = strstr(uri_list, uri_to_search);
+
+	/* Below check is to avoid match for string like "aaa" with "aaas" or "saaa" string present
+	 * in uri_list.
+	 */
+
+	if ((uri_pos != NULL) && ((uri_pos == uri_list) || (!isalnum(*(uri_pos - 1)))) &&
+	    (!isalnum(*(uri_pos + uri_len)))) {
+		is_matched = true;
+	}
+
+	return is_matched;
+}
+
+static void net_buf_put_uri_scheme_list(const struct tbs_inst *inst, struct net_buf_simple *buf)
+{
+	net_buf_simple_reset(buf);
+
+	net_buf_simple_add_mem(buf, inst->uri_scheme_list, strlen(inst->uri_scheme_list));
+
+	/* For GTBS add all the URI scheme list the GTBS bearer has itself, as well as all
+	 * the other TBS bearers
+	 */
+
+	if (inst_is_gtbs(inst)) {
```

Comment:
Consider changing this to
```suggestion
	if (!inst_is_gtbs(inst)) {
		return;
	}
```

To avoid the additional indentation level below